instrumentation/methods/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/methods/MethodInstrumentationModule.java

```diff
@@ -36,32 +35,43 @@ public MethodInstrumentationModule() {
   }
 
   private static List<TypeInstrumentation> createInstrumentations() {
-    InstrumentationConfig config = AgentInstrumentationConfig.get();
-    List<TypeInstrumentation> list =
-        config.isDeclarative()
-            ? MethodsConfig.parseDeclarativeConfig(config.getDeclarativeConfig("methods"))
-            : parseConfigProperties();
```

Comment:
this was a weird case and had to be converted in this initial PR, because now `config.isDeclarative()` is always true